doc/source_tech_preview/deployment/uninstall.rst

```diff
@@ -0,0 +1,13 @@
+Uninstall
+=========
+
+To remove SUSE Containerized OpenStack from your technical preview environment.
+
+From the socok8s directory...
+
+.. code-block::  console
+
+   run.sh remove_deployment
```

Comment:
This doesn't look like a currently supported action, is there a change up to add `remove_deployment` as a supported action?

Reply:
https://github.com/SUSE-Cloud/socok8s/pull/351  adds remove_deployment command for tech preview.